template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabAuthenticationContext.h.ejs

```diff
@@ -16,109 +16,68 @@ class PLAYFABCOMMON_API UPlayFabAuthenticationContext : public UObject
 public:
     UPlayFabAuthenticationContext()
     {
-#ifndef DISABLE_PLAYFABCLIENT_API
         ClientSessionTicket = PlayFabCommon::PlayFabCommonSettings::clientSessionTicket;
-#endif
-#ifndef DISABLE_PLAYFABENTITY_API
         EntityToken = PlayFabCommon::PlayFabCommonSettings::entityToken;
-#endif
         DeveloperSecretKey = PlayFabCommon::PlayFabCommonSettings::developerSecretKey;
-        if(DeveloperSecretKey.Len() != 0)
-        {
-            if(ClientSessionTicket.Len() != 0)
-            {
-                // TODO: Error here appropriately
-                checkf(false, TEXT("Cannot have a ClientSessionTicket with a Developer Secret Key Present!"));
-            }
-        }
-
-        if(ClientSessionTicket.Len() != 0)
-        {
-            if(DeveloperSecretKey.Len() != 0)
-            {
-                // TODO: Error here appropriately
-                checkf(false, TEXT("Cannot have a DeveloperSecretKey with a ClientSessionTicket Present!"));
-            }
-        }
+        checkf(DeveloperSecretKey.Len() != 0 && ClientSessionTicket.Len() != 0, TEXT("Cannot have a ClientSessionTicket with a Developer Secret Key Present!"));
```

Comment:
```suggestion
        checkf(DeveloperSecretKey.len() == 0 || ClientSessionTicket.len() == 0, TEXT("Cannot have a ClientSessionTicket with a Developer Secret Key Present!"));
```

Reply:
Condition inverted here, probably because I had it wrong in my first suggestion (which I corrected in a later suggestion).
Also...
The difference between "Len" and "len" below scares me.
Make sure to thoroughly test this.

Reply:
FString uses .Len() I'll make sure that is aligned.